rollup.config.js

```diff
@@ -1,9 +1,12 @@
 import resolve from '@rollup/plugin-node-resolve';
 import { terser } from "rollup-plugin-terser";
-import { generateSW } from 'rollup-plugin-workbox';
+import { injectManifest } from 'rollup-plugin-workbox';
```

Comment:
the main problem I encountered with the injectManifest function requires that the service worker to already be transpiled and the modules converted, this function just reads the file and injects the precache manifest list.

The plugin doesn't interact with rollup and it ignores the transpiled file in memory. I actually had to find an alternative plugin that uses a lower level api for it to interact properly with rollup, essentially the plugin is a rollup specific rewrite of the webpack workbox plugin.

https://github.com/lee-leonardo/workbox-offline-demo/blob/master/rollup.config.js#L52
https://www.npmjs.com/package/rollup-plugin-workbox-inject

Reply:
Yep, this is the big reason I am more of a fan of your solution actually!